ADR/0009-pipeline-service-via-operator.md

```diff
@@ -0,0 +1,40 @@
+# 0009 Pipeline Service via Operator
+
+Date: 2022-12-01
+
+## Status
+
+Proposed
+
+## Context
+
+kcp is no longer being used as a control plane for StoneSoup. This means that
+"Pipeline Service" cannot be deployed as an independent service. For our
+initial MVP, all Tekton APIs need to be deployed onto a standard OpenShift
+cluster (specifically OpenShift Dedicated).
+
+## Decision
+
+All Tekton APIs will be provided using the stock OpenShift Pipelines operator.
+In the spirit of developing in a "service first" manner, StoneSoup will deploy
+a candidate "nightly" release of the operator. The service will be defined in
+the [pipeline-service](https://github.com/openshift/pipeline-service)
+repository, which is then imported into
+[infra-deployments](https://github.com/redhat-appstudio/infra-deployments) as
+an ArgoCD application.
+
+## Consequences
+
+- Tekton Results will need to be productized and incorporated into the
+OpenShift Pipelines operator.
+- Tekton Triggers should be disabled using appropriate operator configuration.
+- The pruner for Tekton needs to be disabled using appropriate operator configuration.
+- Pipelines as Code should use an appropriate URL to the Hybrid Application
```

Comment:
when you say appropriate, i am guessing you mean a public route accessible from GitHub ?

Reply:
Yes. StoneSoup has an independent interface (Hybrid Application Console) that runs the user experience and lives outside of the cluster. The OpenShift web console won't be accessible from the external internet.